src/app/pages/registro-plan-adquisiciones/components/ver-actividades/ver-actividades.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="container">
+    <div class="row" *ngIf="DatosTransformados">
+        <div class="col px-0">
+            <ngx-general-table [config]="configuracion" [datos]="DatosTransformados"></ngx-general-table>
+        </div>
+    </div>
+</div>
```

Comment:
Revisar si esto se puede reducir como
```suggestion
<ngx-general-table class="container row col px-0"
*ngIf="DatosTransformados" [config]="configuracion" [datos]="DatosTransformados"></ngx-general-table>
```